app/Controller/EventsController.php

```diff
@@ -88,6 +88,12 @@ function edit($id = null) {
                 unset($this->request->data['Event']['finish_time']);
             }
 
+            if(!empty($this->request->data['Event']['deadline_date'])) {
+                $this->request->data['Event']['registration_deadline'] = TimeLib::mysqlDateTimeFormat($this->request->data['Event']['deadline_date'], $this->request->data['Event']['deadline_time']);
```

Comment:
What do you think about defaulting `deadline_time` to 23:59 if none is entered? Right now it defaults to 00:00 which might not be what someone expects if they choose a deadline of today, they mean, end of day today.

Reply:
Yeah, I noticed it as well.  23:59 does make more sense that 0:00.  Included in fixes pull request.